packages/design-system-react-native/src/components/AvatarToken/AvatarToken.constants.ts

```diff
@@ -10,3 +10,14 @@ export const DEFAULT_AVATARTOKEN_PROPS: Required<
   width: '100%',
   height: '100%',
 };
+
+// Sample Token URIs
+export const SAMPLE_AVATARTOKEN_URIS = [
+  'https://cryptologos.cc/logos/ethereum-eth-logo.svg',
+  'https://cryptologos.cc/logos/solana-sol-logo.svg',
+  'https://cryptologos.cc/logos/tether-usdt-logo.svg',
+  'https://cryptologos.cc/logos/cardano-ada-logo.svg',
+  'https://cryptologos.cc/logos/chainlink-link-logo.svg',
+  'https://cryptologos.cc/logos/uniswap-uni-logo.svg',
+  'https://cryptologos.cc/logos/flare-flr-logo.svg',
+];
```

Comment:
Added these so they can be reused for both `AvatarToken`, `AvatarGroup`, and any other components built from `AvatarToken`